### PR TITLE
To Do list: add & remove

### PR DESCRIPTION
**Pull request by jodrack07**
### Learning Objectives:
---
   - Use webpack to bundle JavaScript.
   - Learn how to use proper ES6 syntax.
   - Use ES6 modules to write modular JavaScript.

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -1,7 +1,129 @@
...
+  // put the todo description into the input field
+  taskInput.value = taskDescription;
+};
+
+function deleteTask(id) {
+  // remove one todo from the specified id
+  todos.splice(id, 1);
+  localStorage.setItem('todo-list', JSON.stringify(todos));
+  showTodos();
+}
```
Comment (BenMukebo) [inline src/index.js]: [See this Video](https://www.loom.com/share/bfd464fcf9114111b2f4ed1e67df754d)

- [ ] Great job so far 👏🏻 👏🏻  However as you just saw in the video above ⬆️ , I am unable to remove the book at least after reloading the page, and I must do this every time I need to remove the book . Please correct this.

File: src/index.js [hunk 2]
```diff
@@ -1,7 +1,129 @@
...
+
+// get data from the localstorage
+const todos = JSON.parse(localStorage.getItem('todo-list')) || [];
+
+function showTodos() {
+  taskBox.innerText = '';
+  todos.forEach((todo, index) => {
+    // apply the completed status iff the task/todo is completed
+    const isCompleted = todo.completed ? 'completed' : '';
+    const checked = isCompleted === 'completed' ? 'checked' : '';
```
Comment (BenMukebo) [inline src/index.js]: - [ ] So far, excellent work 👍🏻  however  It's one of the requirements to have a new JavaScript file for the new functionality, kindly create a new JavaScript file that will handle the CRUD (create, update, delete) methods

**Review by BenMukebo — CHANGES_REQUESTED** [review]
### Hi @joseph07-drack,

You did a great job so far 👏🏼 👏🏼
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

### To highlight 💪🏻

- Descriptive Pull Request 👏🏻 
- Professional README :heavy_check_mark: 
- GitHub flow was correctly used.  ✔️   
- A function for adding a new task has been well implemented. :heavy_check_mark: 
- All changes to the To-Do List is saved in local storage., keep rocking :rocket:

 👍  Good job!


## STATUS: CHANGES REQUESTED  ♻️ 
_Please Check the comments under the review and make all required changes_


## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏 💻  
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please ping me @BenMukebo when you comment so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Cathella — APPROVED** [review]
## 🔥 Status: Approved 🟢

### Hi @joseph07-drack 👋🏻, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 


Cheers and Happy coding!👏👏👏

rosycheeks-mochi-peach


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**